### PR TITLE
Delete, view count, ownership

### PR DESCRIPTION
**Pull request by rayolanderos**
Added ownership to the streams for the manage view. The owner will contain the email of the account. 

Added delete functions in manage. 

Added view count to the model for display purposes in Manage. Defauts as 0 and adds a view whenever the single stream is viewed. 

Everytime someone adds a picture, the "date" is updated. 

Has anyone started working on subscriptions? I'm interested in hearing out how this will look in terms of the model. Does anyone have any ideas?

### CODE REVIEW

File: services/manage.py [hunk 1]
```diff
@@ -1,22 +1,35 @@
...
+        
         stream_list = []
 
-        for stream in streams:
-            stream_list.append({
-            'id': stream.key.id(), 
-            'name': stream.name, 
-            'cover_url': stream.cover_url
-            })
+        if streams is not None:
```
Comment (gleono) [inline services/manage.py:23]: No need for the if I think.

**Comment by gleono** [comment]
Looks great!
I was thinking about adding yet another field to the Stream model, with a list of user ids or keys I think might be better.

For subscription do you think that adding send an email to the list of subscribers every time the owner posts a picture. We would need to refactor a little the code from create_stream. It can be reused later for the cron job to send emails to the TAs.

I can add issues to track that and associate it with every pull request, that way our workflow looks cleaner ;)

File: services/manage.py (hunk 1 above)
Comment (rayolanderos) [reply services/manage.py:23]: it errors out privately in the console but it doesn't affect the frontend, it was more for me to control the console errors :)

**Comment by gleono** [comment]
Oh cool, didn’t know that, leave it then. 
		
		

		Descarga Outlook para iOS
	




On Mon, Oct 2, 2017 at 7:34 PM -0500, "rayolanderos" <notifications@github.com> wrote:












@rayolanderos commented on this pull request.





In services/manage.py:
>          stream_list = []
 
-        for stream in streams:
-            stream_list.append({
-            'id': stream.key.id(), 
-            'name': stream.name, 
-            'cover_url': stream.cover_url
-            })
+        if streams is not None:



it errors out privately in the console but it doesn't affect the frontend, it was more for me to control the console errors :)

**Pull request merged**